src/main/java/com/FoodyBuddy/Dao/Impl/OrderDishDAOImpl.java

```diff
@@ -0,0 +1,83 @@
+package com.FoodyBuddy.Dao.Impl;
+
+import java.util.List;
+
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.Transaction;
+import org.springframework.stereotype.Repository;
+
+import com.FoodyBuddy.Dao.OrderDishDAO;
+import com.FoodyBuddy.Model.OrderDish;
+
+@Repository
+public class OrderDishDAOImpl implements OrderDishDAO {
+
+	private SessionFactory sessionFactory;
+	Session session;
+
+    public OrderDishDAOImpl(Session session) {
+        this.session = session;
+    }
+    
+    // saves orderdish object to db
+	public void save(OrderDish c) {
+		Session session = this.sessionFactory.openSession();
```

Comment:
We don't need to create session again. We are getting session from the constructor